### PR TITLE
Fix infinite like glitch

### PR DESCRIPTION
**Pull request by AhmedA1559**
Co-authored-by: Talia Frindell <taliafr@users.noreply.github.com>

Description: …

Tests Performed: …

Trello Card Link: …

### CODE REVIEW

**Comment by shreyasminocha** [comment]
Can we take some inspiration from reddit here and have a `/vote` endpoint that takes an integer as a JSON body field?

"dir indicates the direction of the vote. Voting 1 is an upvote, -1 is a downvote, and 0 is equivalent to "un-voting" by clicking again on a highlighted arrow."

**Comment by AhmedA1559** [comment]
> Can we take some inspiration from reddit here and have a `/vote` endpoint that takes an integer as a JSON body field?

I think the problem here would be that 

> Can we take some inspiration from reddit here and have a `/vote` endpoint that takes an integer as a JSON body field?

Would the latest commit work for this?

**Comment by shreyasminocha** [comment]
Can we get the `dir` from `ctx.request.body` instead? `ctx.params` is for query parameters.

File: backend/src/routes/post.ts [hunk 1]
```diff
@@ -44,19 +45,25 @@ router.post('/:postID/like', async (ctx, next) => {
...
-	ctx.body = post;
-});
-router.post('/:postID/unlike', async (ctx, next) => {
-	const post = await Post.findOneBy(mongodb.ObjectId(ctx.params.postID));
-	if (post === null) {
-		ctx.status = 404;
-		return;
+	// currently, we have no way of identifying based off the request given
+	// so we will just use the username parameter to identify the user
+	if (ctx.request.body.username) {
```
Comment (baylee-rl) [inline backend/src/routes/post.ts]: can we update this now that auth is merged?

Comment (baylee-rl) [reply backend/src/routes/post.ts]: also lint

**Review by baylee-rl — CHANGES_REQUESTED** [review]

**Comment by AhmedA1559** [comment]
Implementation is now in develop.

**Pull request closed without merging**